scripts/validate_packages.py

```
@@ -0,0 +1,267 @@
+        print("\nAvailable packages:", file=sys.stderr)
+        for name in sorted(all_packages.keys()):
+            print(f"  - {name}", file=sys.stderr)
+        sys.exit(1)
+
+    # Check PyPI versions
+    for pkg in validated:
+        pkg_name = pkg["name"]
+        pkg_version = pkg["version"]
+
+        if publish_target == "pypi":
+            result = subprocess.run(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)